packages/template-tachyons/src/Css.ts

```diff
@@ -532,13 +540,45 @@ class CssBuilder<T extends Properties> {
     return this.add("alignItems", "baseline");
   }
   /** Sets `alignItems: "stretch"`. */
-  get ais() {
+  get aiStretch() {
```

Comment:
Hmh! This seems like a pretty big change `ais = start` vs `ais = stretch`. I don't think it'll really have any implications, as stretch and start will both start the content at the beginning of the flex-parent. Though, `stretch` will take up the whole flex-parent, and `start` will only take up the space needed for itself. I would also guess we haven't been using `ais` anywhere, as `align-items: stretch` is the default behavior.

Reply:
Ah shoot, good point! I was just making the `...Stretch` consistent with the other stretch abbreviations we have and didn't realize that, yeah, I was also re-introducing `ais` as something else. :thinking: 

Turns out we do actually use it ~3 times in internal-frontend:

![image](https://user-images.githubusercontent.com/6401/222783825-2d6374e2-064f-4f5e-a78c-c46d67770008.png)

I'm tempted to change anyway, and just update the callers...wydt?

Reply:
Yea, that works for me! Thanks for checking. I'm surprised we set it since that is the default `align-items` value. Maybe worth seeing if its even needed, if you want to. Otherwise, I'm good with just replacing.